{"uid":"1b2a357ce3b86b22","name":"test_TC35_verificar_address_eliminado_despues_de_eliminar_inventory","fullName":"tests.catalog.inventory.test_HU06_EA_DELETE_Inventory#test_TC35_verificar_address_eliminado_despues_de_eliminar_inventory","historyId":"e292ad4a9a207b3e6bc8ae2447df78a6","time":{"start":1755376097139,"stop":1755376097347,"duration":208},"status":"skipped","statusMessage":"XFAIL BUG35: Problemas con la construccion URL de Address\n\nrequests.exceptions.MissingSchema: Invalid URL '/api/v2/admin/inventory-source-addresses/4149': No scheme supplied. Perhaps you meant https:///api/v2/admin/inventory-source-addresses/4149?","statusTrace":"setup_create_inventory = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTUzNzYwNzMsImV4cCI6MTc1NTM3OTY3Mywicm9sZX...ry-sources/still-police', '@type': 'InventorySource', 'address': '/api/v2/admin/inventory-source-addresses/4149', ...})\n\n    @pytest.mark.functional\n    @pytest.mark.high\n    @pytest.mark.xfail(reason=\"BUG35: Problemas con la construccion URL de Address\", run=True)\n    def test_TC35_verificar_address_eliminado_despues_de_eliminar_inventory(setup_create_inventory):\n        headers, inventory = setup_create_inventory\n        url_inventory = EndpointInventory.code(inventory[\"code\"])\n        address_url = inventory.get(\"address\")\n        response_delete = SyliusRequest.delete(url_inventory, headers)\n        AssertionStatusCode.assert_status_code_204(response_delete)\n        log_request_response(url_inventory, response_delete, headers, payload=inventory)\n>       response_get_address = SyliusRequest.get(address_url, headers)\n                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/catalog/inventory/test_HU06_EA_DELETE_Inventory.py:66: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc/routes/request.py:7: in get\n    response = requests.get(url, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/api.py:73: in get\n    return request(\"get\", url, params=params, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/api.py:59: in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/sessions.py:575: in request\n    prep = self.prepare_request(req)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/sessions.py:484: in prepare_request\n    p.prepare(\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/models.py:367: in prepare\n    self.prepare_url(url, params)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <PreparedRequest [GET]>\nurl = '/api/v2/admin/inventory-source-addresses/4149', params = OrderedDict()\n\n    def prepare_url(self, url, params):\n        \"\"\"Prepares the given HTTP URL.\"\"\"\n        #: Accept objects that have string representations.\n        #: We're unable to blindly call unicode/str functions\n        #: as this will include the bytestring indicator (b'')\n        #: on python 3.x.\n        #: https://github.com/psf/requests/pull/2238\n        if isinstance(url, bytes):\n            url = url.decode(\"utf8\")\n        else:\n            url = str(url)\n    \n        # Remove leading whitespaces from url\n        url = url.lstrip()\n    \n        # Don't do any URL preparation for non-HTTP schemes like `mailto`,\n        # `data` etc to work around exceptions from `url_parse`, which\n        # handles RFC 3986 only.\n        if \":\" in url and not url.lower().startswith(\"http\"):\n            self.url = url\n            return\n    \n        # Support for unicode domain names and paths.\n        try:\n            scheme, auth, host, port, path, query, fragment = parse_url(url)\n        except LocationParseError as e:\n            raise InvalidURL(*e.args)\n    \n        if not scheme:\n>           raise MissingSchema(\n                f\"Invalid URL {url!r}: No scheme supplied. \"\n                f\"Perhaps you meant https://{url}?\"\n            )\nE           requests.exceptions.MissingSchema: Invalid URL '/api/v2/admin/inventory-source-addresses/4149': No scheme supplied. Perhaps you meant https:///api/v2/admin/inventory-source-addresses/4149?\n\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/models.py:438: MissingSchema","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1755376072485,"stop":1755376072486,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"auth_headers","time":{"start":1755376072486,"stop":1755376073195,"duration":709},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_create_inventory","time":{"start":1755376096909,"stop":1755376097138,"duration":229},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL BUG35: Problemas con la construccion URL de Address\n\nrequests.exceptions.MissingSchema: Invalid URL '/api/v2/admin/inventory-source-addresses/4149': No scheme supplied. Perhaps you meant https:///api/v2/admin/inventory-source-addresses/4149?","statusTrace":"setup_create_inventory = ({'Authorization': 'Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpYXQiOjE3NTUzNzYwNzMsImV4cCI6MTc1NTM3OTY3Mywicm9sZX...ry-sources/still-police', '@type': 'InventorySource', 'address': '/api/v2/admin/inventory-source-addresses/4149', ...})\n\n    @pytest.mark.functional\n    @pytest.mark.high\n    @pytest.mark.xfail(reason=\"BUG35: Problemas con la construccion URL de Address\", run=True)\n    def test_TC35_verificar_address_eliminado_despues_de_eliminar_inventory(setup_create_inventory):\n        headers, inventory = setup_create_inventory\n        url_inventory = EndpointInventory.code(inventory[\"code\"])\n        address_url = inventory.get(\"address\")\n        response_delete = SyliusRequest.delete(url_inventory, headers)\n        AssertionStatusCode.assert_status_code_204(response_delete)\n        log_request_response(url_inventory, response_delete, headers, payload=inventory)\n>       response_get_address = SyliusRequest.get(address_url, headers)\n                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\ntests/catalog/inventory/test_HU06_EA_DELETE_Inventory.py:66: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nsrc/routes/request.py:7: in get\n    response = requests.get(url, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/api.py:73: in get\n    return request(\"get\", url, params=params, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/api.py:59: in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/sessions.py:575: in request\n    prep = self.prepare_request(req)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/sessions.py:484: in prepare_request\n    p.prepare(\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/models.py:367: in prepare\n    self.prepare_url(url, params)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <PreparedRequest [GET]>\nurl = '/api/v2/admin/inventory-source-addresses/4149', params = OrderedDict()\n\n    def prepare_url(self, url, params):\n        \"\"\"Prepares the given HTTP URL.\"\"\"\n        #: Accept objects that have string representations.\n        #: We're unable to blindly call unicode/str functions\n        #: as this will include the bytestring indicator (b'')\n        #: on python 3.x.\n        #: https://github.com/psf/requests/pull/2238\n        if isinstance(url, bytes):\n            url = url.decode(\"utf8\")\n        else:\n            url = str(url)\n    \n        # Remove leading whitespaces from url\n        url = url.lstrip()\n    \n        # Don't do any URL preparation for non-HTTP schemes like `mailto`,\n        # `data` etc to work around exceptions from `url_parse`, which\n        # handles RFC 3986 only.\n        if \":\" in url and not url.lower().startswith(\"http\"):\n            self.url = url\n            return\n    \n        # Support for unicode domain names and paths.\n        try:\n            scheme, auth, host, port, path, query, fragment = parse_url(url)\n        except LocationParseError as e:\n            raise InvalidURL(*e.args)\n    \n        if not scheme:\n>           raise MissingSchema(\n                f\"Invalid URL {url!r}: No scheme supplied. \"\n                f\"Perhaps you meant https://{url}?\"\n            )\nE           requests.exceptions.MissingSchema: Invalid URL '/api/v2/admin/inventory-source-addresses/4149': No scheme supplied. Perhaps you meant https:///api/v2/admin/inventory-source-addresses/4149?\n\n/opt/hostedtoolcache/Python/3.13.6/x64/lib/python3.13/site-packages/requests/models.py:438: MissingSchema","steps":[],"attachments":[{"uid":"8fff5d8eb361e330","name":"log","source":"8fff5d8eb361e330.txt","type":"text/plain","size":2282}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true},"afterStages":[{"name":"setup_create_inventory::0","time":{"start":1755376097377,"stop":1755376097895,"duration":518},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"high"},{"name":"tag","value":"functional"},{"name":"parentSuite","value":"tests.catalog.inventory"},{"name":"suite","value":"test_HU06_EA_DELETE_Inventory"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2404-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.catalog.inventory.test_HU06_EA_DELETE_Inventory"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":2,"passed":0,"unknown":0,"total":2},"items":[{"uid":"56fba4155afe7145","reportUrl":"https://Elv500.github.io/Softsign/35//#testresult/56fba4155afe7145","status":"skipped","statusDetails":"XFAIL BUG35: Problemas con la construccion URL de Address\n\nrequests.exceptions.MissingSchema: Invalid URL '/api/v2/admin/inventory-source-addresses/4102': No scheme supplied. Perhaps you meant https:///api/v2/admin/inventory-source-addresses/4102?","time":{"start":1755374677252,"stop":1755374677465,"duration":213}}]},"tags":["high","functional"]},"source":"1b2a357ce3b86b22.json","parameterValues":[]}